framework/include/dirackernels/DiracKernelInfo.h

```diff
@@ -78,7 +79,8 @@ class DiracKernelInfo
    * Used by client DiracKernel classes to determine the Elem in which
    * the Point p resides.  Uses the PointLocator owned by this object.
    */
-  const Elem * findPoint(Point p, const MooseMesh & mesh);
+  const Elem *
+  findPoint(Point p, const MooseMesh & mesh, const std::set<SubdomainID> * blocks = nullptr);
```

Comment:
There should be no need for us to modify `p` and consequently not really any desire to copy construct another `Point`, so while we're in here, let's go ahead and change this to `const Point & p`